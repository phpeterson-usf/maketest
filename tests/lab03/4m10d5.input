-e "4 * (10 / 5)"
