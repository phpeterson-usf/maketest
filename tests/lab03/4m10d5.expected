EXPR OPER2 MULT
..EXPR INTVAL 4
..EXPR OPER2 DIV
....EXPR INTVAL 10
....EXPR INTVAL 5
